version/6_14_4.sh

```diff
@@ -1,7 +1,7 @@
 #!/bin/bash -e
 
-NODE_VERSION=6.14.3
+NODE_VERSION=6.14.4
 echo "=============== Installing Node $NODE_VERSION ============="
 . /root/.nvm/nvm.sh && nvm install $NODE_VERSION
-nvm use $NODE_VERSION && npm install npm@6.3.0 -g
+nvm use $NODE_VERSION && npm install npm@6.4.0 -g
```

Comment:
Same here, the way we define the version should be consistent across these images.